plenum/cli/cli.py

```diff
@@ -78,7 +79,7 @@
 from plenum.common.log import CliHandler, getlogger, Logger, \
     getRAETLogLevelFromConfig, getRAETLogFilePath, TRACE_LOG_LEVEL
 from plenum.server.node import Node
-from plenum.common.types import CLIENT_STACK_SUFFIX, NodeDetail, HA
+from plenum.common.types import NodeDetail, HA
```

Comment:
I *really* dislike the name "HA" here. It's a code smell. We need to rename it something like "HostAddress". That doesn't necessarily need to be done in this pull request, though...